lib/bloc/user_profile/user_profile_bloc.dart

```diff
@@ -240,6 +245,9 @@ class UserProfileBloc {
 
   void _listenRegistrationRequests(ServiceInjector injector) {
     _registrationController.stream.listen((request) async {
+      await _updateEnrolledBiometrics();
+      _updateSecurityModel(UpdateSecurityModel(SecurityModel.initial().copyWith(
```

Comment:
This is problematic.
Moving the reset security model to the registration point creates a side effect where it is also happens after restore. It will reset the PIN and backup mnemonics that have been assigned right after the restore.
We should keep the reset only from the "Let's Breez" button. If necessary we can implement an action only for that (ResetSecurityModel).